main.tf

```diff
@@ -186,6 +186,15 @@ resource "aws_s3_bucket" "default" {
             replica_kms_key_id = try(rules.value.destination.replica_kms_key_id, null)
             account_id         = try(rules.value.destination.account_id, null)
 
+            # https://docs.aws.amazon.com/AmazonS3/latest/userguide/replication-walkthrough-5.html
+            dynamic "metrics" {
+              for_each = try(rules.value.destination.metrics.status, "") == "Enabled" ? [1] : []
+
+              content {
+                status = "Enabled"
+              }
```

Comment:
We should also add support for `metrics.minutes` https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/s3_bucket#minutes

Reply:
Only "15" is supported currently, I'd hardcode